spec/api/harvester.yaml

```
@@ -8,14 +8,14 @@ servers:
   - url: http://localhost:32308/
 
 paths:
-  /FederationRelationship:
+  /federation-relationship:
```

let's use the plural form for all resources
```suggestion
  /federation-relationships:
```